source/agora/consensus/EnrollmentManager.d

```diff
@@ -457,72 +450,65 @@ public class EnrollmentManager
 unittest
 {
     import agora.common.Amount;
-    import agora.common.Config;
-    import agora.common.TransactionPool;
-    import agora.consensus.data.Block;
     import agora.consensus.data.Transaction;
-    import agora.consensus.data.UTXOSet;
     import agora.consensus.Genesis;
-    import agora.node.BlockStorage;
     import std.format;
     import std.conv;
 
+    Transaction[Hash] storage;
+    scope findUTXO = (Hash hash, size_t index, out UTXOSetValue value) @trusted
```

Comment:
Maybe we could factor this out into a small utility, it seems to be a common need ?

Reply:
True. I'll take a look soon if we can combine all of them.